src/main/java/at/hannibal2/skyhanni/utils/RegistryUtil.kt

```diff
@@ -0,0 +1,56 @@
+package at.hannibal2.skyhanni.utils
+
+object RegistryUtil {
+
+    private const val PREFIX: String = "REG"
+    private const val SUCCESS_CODE: Int = 0
+    private const val FAILURE_CODE: Int = 1
+
+    fun addKey(runtime: Runtime, location: RegistryPaths, key: String) : Boolean {
+        val proc: Process = runtime.exec("$PREFIX ADD ${location.path}\\$key")
```

Comment:
Maybe we can skip some shlexing here and use the `Runtime#exec(String[])` variant instead.